CODE_OF_CONDUCT.rst

```diff
@@ -0,0 +1,71 @@
+and any other forums of the project. In addition, violations of this
+code outside these spaces may affect a person's ability to participate
+within them.
+
+If you believe someone is violating the code of conduct, we ask that
+you report it by emailing the maintainers listed in the README.
+
+-  Be friendly and patient.
+-  Be welcoming. We strive to be a community that welcomes and
+   supports people of all backgrounds and identities. This includes,
+   but is not limited to members of any race, ethnicity, culture,
+   national origin, color, immigration status, social and economic
+   class, educational level, sex, sexual orientation, gender identity
+   and expression, age, size, family status, political belief,
+   religion, and mental and physical ability.
+-  Be considerate. Your work will be used by other people, and you in
+   turn will depend on the work of others. Any decision you take will
+   affect users and colleagues, and you should take those consequences
+   into account when making decisions.
+-  Be respectful. Not all of us will agree all the time, but
+   disagreement is no excuse for poor behavior and poor manners. We
+   might all experience some frustration now and then, but we cannot
+   allow that frustration to turn into a personal attack. It is
+   important to remember that a community where people feel
+   uncomfortable or threatened is not a productive one. Everyone
+   involved in the project should be respectful when dealing with
+   others.
+-  Be careful in the words that you choose. We want to be a community
+   of professionals, and we conduct ourselves professionally. Be kind
+   to others. Do not insult or put down other participants. Harassment
+   and other exclusionary behavior are not acceptable. This includes,
+   but is not limited to:
+
+   -  Violent threats or language directed against another person.
+   -  Discriminatory jokes and language.
+   -  Posting sexually explicit or violent material.
+   -  Posting (or threatening to post) other people's personally
+      identifying information ("doxing").
+   -  Personal insults, especially those using racist or sexist terms.
+   -  Unwelcome sexual attention.
+   -  Advocating for, or encouraging, any of the above behavior.
+   -  Repeated harassment of others. In general, if someone asks you
+      to stop, then stop.
+
+-  When others disagree, try to understand why. Disagreements, both social
+   and technical, happen all the time and this project is no exception. It
+   is important that we resolve disagreements and differing views
+   constructively. Different people have different perspectives on
+   issues. Being unable to understand why someone holds a viewpoint
+   does not mean that they are wrong. Do not forget that it is human to
+   err and blaming each other does not get us anywhere. Instead, focus
+   on helping to resolve issues and learning from mistakes.
+
+This Code of Conduct is adapted from the `Django Code of Conduct
```

Comment:
I think we need to mention here that this version is also under CC BY 4.0 (I think a mix of CC 3.0 and 4.0 can be 4.0)

Reply:
done